samplemanager/sample_manager.py

```diff
@@ -80,12 +81,14 @@ def run(self):
                 exit()
 
     def finding_and_adding_sample(self):
+        instance = questionary.text("Enter the DAS instance for the search", style=custom_style, default=self.default_instance).ask()
```

Comment:
I think it would be a good idea to add a bit more information on what DAS instances mean here and where to find the relevant information (Same with DAS nick tbh). The [readthedocs](https://crown.readthedocs.io/en/latest/kingmaker.html#addition-of-new-samples) site has some information on the nicks, but none on the instances so far.

Reply:
Appears now at the docs: https://crown.readthedocs.io/en/latest/kingmaker.html#information-on-cms-datasets